tasks/htmltasks.js

```diff
@@ -32,6 +32,18 @@ gulp.task('html:lint', () => {
     .pipe(gulpTsLint.report())
 });
 
+// Compile protocol for server and client communication
```

Comment:
We should be able to get rid of this entire tasks file. Ideally we will compile the entire project at once. we will still need a separate task to webpack the webview application separately but that is a separate issue.